### ISSUE TITLE
Icon Request: Operating Systems

### ISSUE THREAD

**Issue by dags170291** [colebemis/feather#184]
Can you please do icons for various operating systems? Eg. Linux, Android, IOS, Windows
<img width="455" alt="screen shot 2017-09-10 at 1 35 07 pm copy 2" src="https://user-images.githubusercontent.com/6403926/30252916-4c3e8d14-9627-11e7-971f-583499618abd.png">

**Comment by emresaracoglu** [comment]
I need too 😄

**Comment by johnletey** [comment]
@colebemis @locness3 @dags170291 @emresaracoglu How does [this](https://www.figma.com/file/gzFMbKYZeXYrA8gCgXOnBt/os-icons) look?

**Comment by emresaracoglu** [comment]
So good 👍@johnletey

**Comment by johnletey** [comment]
Wonderful @emresaracoglu ... glad you like them!

**Comment by locness3** [comment]
> @colebemis @locness3 @dags170291 @emresaracoglu How does [this](https://www.figma.com/file/gzFMbKYZeXYrA8gCgXOnBt/os-icons) look?

As I said in another issue, please follow #171.

**Comment by johnletey** [comment]
What does everyone think of my Ubuntu icon? [Figma](https://www.figma.com/file/C7cEHvYfSsGpvx9utbnFnhJy/Feather-Icons?node-id=4%3A2)

@locness3 @ahtohbi4

![Screen Shot 2019-07-08 at 7 23 48 AM](https://user-images.githubusercontent.com/30328854/60783124-f23cbb00-a151-11e9-8214-0922533fc495.png)

**Comment by locness3** [comment]
That's great! Maybe breaking the circle to make it more like the original ?

**Comment by johnletey** [comment]
@locness3 Not exactly quite sure how to do that ... have any tips @ahtohbi4?

**Comment by ahtohbi4** [comment]
@johnletey something like this:

<img width="453" alt="Снимок экрана 2019-07-10 в 12 37 35" src="https://user-images.githubusercontent.com/3686790/60959124-54dbb580-a310-11e9-86b1-8767e6d7aaa8.png">

<img width="506" alt="Снимок экрана 2019-07-10 в 12 37 11" src="https://user-images.githubusercontent.com/3686790/60959133-57d6a600-a310-11e9-9e9b-4cc867e4ad7e.png">

But I don't like spaces... I will think about it. 🤔

**Comment by locness3** [comment]
That doesn't remind me of Ubuntu to be honest... Maybe not making the circle inside at the outside dots

**Comment by johnletey** [comment]
Yes @ahtohbi4 ... I aggree with @locness3 ... that doesn't remind me at all of the Ubuntu icon!

**Comment by johnletey** [comment]
What does everyone think of my Apple icon? [Figma](https://www.figma.com/file/C7cEHvYfSsGpvx9utbnFnhJy/Feather-Icons?node-id=68%3A24)

cc. @locness3 @ahtohbi4

![Shot 2019-07-21 at 16 41 45](https://user-images.githubusercontent.com/30328854/61593370-956fe480-abd6-11e9-9da8-5eb7a096865a.jpg)

**Comment by locness3** [comment]
Great!

**Comment by johnletey** [comment]
Wonderful ... I'll go ahead and submit a PR!

**Comment by johnletey** [comment]
Hey everyone, how does my Ubuntu and Android icons look?

@locness3 For the Ubuntu icon, it was too complex to cut the inner circle into three chunks ... as we use rounded line caps, so no matter how I cut the circle, it cuts weren't quite apparent

cc. @ahtohbi4 @mittalyashu

![Shot 2019-07-23 at 07 54 07](https://user-images.githubusercontent.com/30328854/61689572-5d27ed80-ad1f-11e9-8d0a-6f277448d16d.jpg)

**Comment by ahtohbi4** [comment]
@johnletey #171 disallows to use 1px stroke width, as you do for small circles of Ubuntu icon.

¯\\\_(ツ)_/¯

**Comment by mittalyashu** [comment]
Hey @johnletey,

Can you try to increase the head height by 1px, just to add more spacing?

Since you have already created the windows icon, can you also create a Microsoft icon too?

**Comment by johnletey** [comment]
> @johnletey #171 disallows to use 1px stroke width, as you do for small circles of Ubuntu icon.

@ahtohbi4 Those small circles are 1x1px but with a 2px stroke centred

> Can you try to increase the head height by 1px, just to add more spacing?

@mittalyashu Unfortunately, when I do this the icon gets disproportioned and thus looks weird!

> Since you have already created the windows icon, can you also create a Microsoft icon too?

Like this? [Figma](https://www.figma.com/file/C7cEHvYfSsGpvx9utbnFnhJy/Feather-Icons?node-id=126%3A7)

![Shot 2019-07-23 at 10 52 33](https://user-images.githubusercontent.com/30328854/61702834-607ba300-ad38-11e9-9d5d-01d0a25addae.jpg)

![Shot 2019-07-23 at 10 55 07](https://user-images.githubusercontent.com/30328854/61702839-61acd000-ad38-11e9-89c6-13e631df244e.jpg)

**Comment by mittalyashu** [comment]
Microsoft & Grid icon looks exactly the same.

![image](https://user-images.githubusercontent.com/29014463/61705596-b0bc2a80-ad63-11e9-8bfc-6b8e949f8254.png)

// courtesy @johnletey

**Comment by locness3** [comment]
So, I think we don't need a microsoft icon @mittalyashu .

**Comment by johnletey** [comment]
@locness3 I agree
